CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing
+
+Thanks for being willing to contribute!
+
+**Working on your first Pull Request?** You can learn how from this _free_
+series [How to Contribute to an Open Source Project on GitHub][egghead]
+
+## Project setup
+
+1.  Fork and clone the repo
+2.  Run `npm run setup` to install dependencies and run validations.
```

Comment:
I'm not sure if the validations bit is necessary after pulling dependencies and just starting out, as they aren't really set up to watch for changes. Perhaps mentioning how to run tests in watch mode or starting up a dev environment in storybook might be useful?

Reply:
I've seen this practice before. It may also be beneficial for potential contributors to have this sort of sanity check at the beginning where they see they receive a thing that passes all its validations. Mentioning how all the other things are run is already covered in the README.